### ISSUE TITLE
[No Interactive Divs] User profile buttons

### ISSUE THREAD

**Issue by AccessiT3ch** [Tomerozfresko/groupomania#2]
@Tomerozfresko - Right now the functionality to update user profile and/or logout is inaccessible via keyboard navigation.  A lot of sighted and non-sighted users utilize the keyboard to interact with sites, so we don't want to make any critical functionality inaccessible to them. This largely comes down to making things ta bbable, which means interactive elements need to be focusable, which is only available for certain (interactive) HTML elements.

It is really common for web devs to add event listeners to divs, but since divs cannot be focused upon, they are semantically incorrect. In this case, the interactive elements should probably either be `a` or `button` elements.

<img width="709" alt="Screenshot 2022-11-15 at 9 23 24 AM" src="https://user-images.githubusercontent.com/95449364/201985529-4159c713-ca99-4f36-a3c2-5f4216a0b7eb.png">

**Comment by AccessiT3ch** [comment]
This should also be fixed for the trashcan icon when deleting a post, and the like and comments buttons.

**Comment by Tomerozfresko** [comment]
@AccessiT3ch Done in commit [9ede6e9](https://github.com/Tomerozfresko/groupomania/commit/9ede6e9fb7bd7435db0c94aefe01e9e211f037d8)
